spec/controllers/home_controller_spec.rb

```diff
@@ -4,9 +4,60 @@
   render_views
 
   describe 'GET #index' do
-    it 'redirects to about page' do
-      get :index
-      expect(response).to redirect_to(about_path)
+    context 'when not signed in' do
+      it 'redirects to about page' do
+        get :index
+        expect(response).to redirect_to(about_path)
+      end
+    end
+
+    context 'when signed in' do
+      let(:user) { Fabricate(:user) }
+      subject do
+        sign_in(user)
+        get :index
+      end
+
+      it 'assigns @body_classes' do
+        subject
+        expect(assigns(:body_classes)).to eq 'app-body'
```

Comment:
I have been merging these (not having heard of the rationale in that issue) so at this point it would inconsistent to reject this one. I suppose it's not horrible if we merge it and then refactor it to be better tests later? Coverage is coverage, even if it covers a bit too much detail.